### ISSUE TITLE
cogvideox-5b生成视频是噪声

### ISSUE THREAD

**Issue by Jake-wei** [THUDM/CogVideo#215]
### System Info / 系統信息

H800显卡，diffusers 版本是0.31.0.dev

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

import torch
from diffusers import CogVideoXPipeline,CogVideoXDPMScheduler
from diffusers.utils import export_to_video
from modelscope import snapshot_download
import os
# os.environ["CUDA_VISIBLE_DEVICES"] = "6"
model_dir = snapshot_download(
            "ZhipuAI/CogVideoX-5b",
            ignore_patterns=["*.h5"],  # 忽略下载哪些文件
            local_dir="ZhipuAI/CogVideoX-5b")
prompt = "A panda, dressed in a small, red jacket and a tiny hat."

pipe = CogVideoXPipeline.from_pretrained(
    "ZhipuAI/CogVideoX-5b",
    torch_dtype=torch.bfloat16
)
pipe.scheduler = CogVideoXDPMScheduler.from_config(pipe.scheduler.config, timestep_spacing="trailing")
#  这行解决了cublas的问题
torch.cuda.set_device(1)
pipe.to("cuda:1")

#pipe.enable_model_cpu_offload()
#pipe.enable_sequential_cpu_offload()
#pipe.vae.enable_slicing()
#pipe.vae.enable_tiling()
video = pipe(
    prompt=prompt,
    num_videos_per_prompt=1,
    num_inference_steps=50,
    num_frames=49,
    guidance_scale=6,
    generator=torch.Generator(device="cuda").manual_seed(42),
    use_dynamic_cfg=True,
).frames[0]

export_to_video(video, "output.mp4", fps=8)

### Expected behavior / 期待表现

这个几乎就是从官方copy的示例，但是生成效果是噪声

https://github.com/user-attachments/assets/5bed9273-2482-4946-9d05-f3e71d034e49

**Comment by zRzRzRzRzRzRzR** [comment]
稳定复现吗，我一次都复现不出来
"A panda, dressed in a small, red jacket and a tiny hat."
这个提示词太短了，请查看我们的demo中，必须使用长提示词，因此你需要使用GLM-4 或者 GPT-4润色
其他长提示词是否复现，此外，你是否用了任何量化

**Comment by Jake-wei** [comment]
> 稳定复现吗，我一次都复现不出来 "A panda, dressed in a small, red jacket and a tiny hat." 这个提示词太短了，请查看我们的demo中，必须使用长提示词，因此你需要使用GLM-4 或者 GPT-4润色 其他长提示词是否复现，此外，你是否用了任何量化

当prompt不变的时候，稳定复现。模型没有量化，就是从modelscope下载的5B版本模型，就如代码中所写的bf16精度，但是在推理时，终端日志打印了 ：
The config attributes {'use_rotary_positional_embeddings': True} were passed to CogVideoXTransformer3DModel, but are not expected and will be ignored. Please verify your config.json configuration file.

我猜测是这个原因吗

**Comment by zRzRzRzRzRzRzR** [comment]
use_rotary_positional_embeddings这个就是你没正常更新diffusers0.30.1重新安装就行了

**Comment by Jake-wei** [comment]
> use_rotary_positional_embeddings这个就是你没正常更新diffusers0.30.1重新安装就行了

我试一下卸载重装，看来虽然版本显示是0.31.0，但是没正常更新版本。辛苦解答 ，感谢！
